static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -305,10 +309,6 @@ class TeamMembers extends AsyncView<Props, State> {
   }
 
   render() {
-    if (this.state.loading) {
-      return <LoadingIndicator />;
-    }
-
```

Comment:
if we have a lot of members, won't we need this loading indicator?

Reply:
Maybe not. There's another loading indicator that loads on the main page. So basically first time you load all the members there's a loading indicator, this one shows up when you add or remove a member and you're not loading all team members in that case. My thought is that it's not necessary since it's always a quick action.

Reply:
@cathteng what's the best way to test that?

Reply:
if you mean locally, i think you can create a bunch of members in your organization and see what opening the dropdown looks like

if you mean in testing, i know you can mock the API response for the members of the organization, but i'm not sure that making a long list of members will cause the dropdown's API call to be slow enough to show the loading indicator

Reply:
The loading state is already handled by `AsyncView` and `dropdownBusy`. Removing this is fine.

Reply:
I ended up testing this locally at least so here's a demo. It's not the real life experience and I don't it's reasonable to make a complicated test like that so I stick with local.

Uploading Screen Recording 2023-05-15 at 4.08.04 PM.mov…